recipe/meta.yaml

```diff
@@ -93,9 +89,11 @@ outputs:
       - lib/libblas.dylib                       # [osx]
       - lib/libblas.{{ version_major }}.dylib   # [osx]
       - lib/libblas.{{ version }}.dylib         # [osx]
-      - Library/bin/libblas.dll                 # [win]
+      - Library/bin/blas.dll                    # [win]
```

Comment:
This is a ABI breaking incompatibility.

Reply:
For eg: you won't be able to import numpy in the scipy recipe.

Reply:
I had commented on this further up
> AFAIU the name change of the libraries should not matter, because the only place where we really rely on those is for building/linking the tests on the blas feedstock.

... since we just [copy](https://github.com/conda-forge/blas-feedstock/blob/main/recipe/build_pkg.bat) the implementation DLL.

Could you let me know which way forward you'd prefer?
* rename the DLLs here to keep the prior name
* only switch to flang for lapack 3.11+
* something else

Reply:
> AIU the name change of the libraries should not matter, because the only place where we really rely on those is for building/linking the tests on the blas feedstock.

They are used when we do `libblas=*=*netlib`. With the change here `conda install numpy libblas=*=*netlib` will fail to load numpy.

> Could you let me know which way forward you'd prefer?

Rename the DLL and fix the import library to refer to the renamed DLL name